CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## v1.0.0
 
 ### General
+- Code has been made python 3 compatible. [[#804]](https://github.com/aiidateam/aiida_core/pull/804)
```

Comment:
What about mentioning the fact that we now make sure everything is written in UTF-8?

Reply:
I don't know, I think it becomes a bit technical (but ok if you want to add it). Rather, is it possible to add a link to all the PRs related to Python 3? The issue you link is actually just the tracker. The other option is just to link the other issues in the tracker issue

Reply:
Good point, will add that PR

Reply:
There is quite a number of individual PR's not all of them interesting per se, that's why I did not add them all individually but chose to use the tracker, which was used as well as the central issue for the long term project. Which PRs would you like to link?

Reply:
Just saying that if I click on the issue I don't have an easy way to check the code changes. If it is possible just to have a list like "addressed in #xxx, #yyy, ..." better, otherwise don't worry.

Reply:
Done

Reply:
I haven't explicitly added the utf-8 PR but is included in the python 3 list